__tests__/app.test.js

```diff
@@ -79,4 +80,73 @@ describe('Express app',() => {
 
 
     });
+
+    describe.only('PATCH /api/articles/:article_id',() => {
+        it('200: /api/articles/1 when article_id exists',() => {
+            return request(app).patch('/api/articles/1').query({inc_votes:1}).expect(200);
+        });
+
+        it('200: /api/articles/1 returns the article with article_id=1',() => {
+            const patchPromise=request(app).patch('/api/articles/1').query({inc_votes:1});
+            const getPromise=request(app).get('/api/articles/1');
+
+            return Promise.all([patchPromise,getPromise]).then(([patchResult,getResult])=>{
+                expect(patchResult.body).toEqual(getResult.body);
+            });
+        });
+
+        it('200: /api/articles/1 returns article 1, updated as per the PATCH body',() => {
+            const getPromise=request(app).get('/api/articles/1');
+            const patchPromise=request(app).patch('/api/articles/1').query({inc_votes:10});
+
+            return Promise.all([getPromise,patchPromise]).then(([getResult,patchResult])=>{
+                expect(patchResult.body.article.votes).toBe(getResult.body.article.votes + 10);
+            });
+        });
+
+        it('200: inc_votes in the PATCH body is a negative number',() => {
+            const getPromise=request(app).get('/api/articles/1');
+            const patchPromise=request(app).patch('/api/articles/1').query({inc_votes:-30});
+
+            return Promise.all([getPromise,patchPromise]).then(([getResult,patchResult])=>{
+                expect(patchResult.body.article.votes).toBe(getResult.body.article.votes -30);
+            });
+        });
+
+        it('400: Empty PATCH body returns an "Invalid PATCH body" error',() => {
+            return request(app).patch('/api/articles/1').expect(400).then(({body})=>{
+                expect(body).toEqual({msg:'Invalid PATCH body'});
+            })
+        });
+
+        it('400: Extra fields in PATCH body returns "Invalid PATCH body" error',() => {
+            return request(app).patch('/api/articles/1').query({inc_votes:10,body:'Removed'}).expect(400).then(({body})=>{
```

Comment:
If there are unnecessary properties inside the patch body, you just want to ignore them. As long as `inc_votes` is there, should be ok!